src/CRM/CivixBundle/Resources/views/Code/phpunit-boot-cv.php.php

```diff
@@ -27,11 +27,11 @@
  * @throws \RuntimeException
  *   If the command terminates abnormally.
  */
-function cv($cmd, $decode = 'json') {
+function cv(string $cmd, $decode = 'json'): string {
```

Comment:
The return type here is not correct -- if the `$decode` mode is `raw` or `phpcode`, then it will return `string`. But if it's `json` mode, then it can be any JSON-style value. Pushed a correction.